src/assets/thirdPartyServices/index.js

```diff
@@ -162,7 +164,7 @@ export default class SampleServices {
     if (typeof component === "undefined") {
       component = this.serviceOrgIDToComponent[this.generateUniqueID(orgId, serviceId, defaultChainId)];
       if (typeof component === "undefined") {
-        component = DefaultService;
+        component = <ErrorMsgBox showErr errorMsg="No Component matched" />;
```

Comment:
Are you sure we want to show an error message here? I thought the idea was to show some default UI in case the Service developer does not provide CustomUI.

Reply:
We're not providing default UI anymore.. that's y used the ErrMessage here.